src/Optimizely/Event/Builder/EventBuilder.php

```diff
@@ -38,14 +41,10 @@ class EventBuilder
     const SDK_VERSION = '1.2.0';
 
     /**
-     * @var string URL to send impression event to.
-     */
-    private static $IMPRESSION_ENDPOINT = 'https://logx.optimizely.com/log/decision';
-
-    /**
-     * @var string URL to send conversion event to.
+     * @var string URL to send 
+     * event to.
```

Comment:
Please copy from C# SDK fro end point.

Reply:
Since both Impression/Conversion End Point URLs are now same, there is no need to keep separate identifiers. $END_POINT is already updated with new URL